common/component/kafka/kafka.go

```diff
@@ -263,6 +264,11 @@ func getSchemaSubject(topic string) string {
 }
 
 func (k *Kafka) DeserializeValue(message *sarama.ConsumerMessage, config SubscriptionHandlerConfig) ([]byte, error) {
+	// Null Data is valid and a tombstone record. It shouldn't be serialized
+	if message.Value == nil {
+		return json.Marshal(nil)
```

Comment:
Since this is a constant value...

```suggestion
		return []byte("null"), nil;
```

Reply:
I don't think these 2 are the same.. I want to return NULL not the string "null"...

Reply:
> json.Marshal(nil) produces a proper JSON null value, while []byte("null") is just a byte slice containing the string “null”

Reply:
https://go.dev/play/p/Tj_PlstjW7K
Note that `uint8 == byte` (they are aliases)

Reply:
You're right. Validated it works. That will teach me to teach an AI instead of a human ;)